src/components/mapView/MapView.jsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import "./MapView.css";
+import mapViewSamleImg from "./../../../public/map-view-samle-img.png";
```

Comment:
Also a small one - typo- I guess you meant to write map-view-sample-img.png, not map-view-samle-img.png

Reply:
Thank you for your review! Both comments have been addressed.